extension/code-editor-integration/README.md

```diff
@@ -1,4 +1,5 @@
 # Code Editor Integration Development
+![Node.js CI](https://github.com/gleitz/howdoi/workflows/Node.js%20CI/badge.svg)
```

Comment:
I'm all about shields.io now because you can change the colors (!)

Try `https://img.shields.io/github/workflow/status/gleitz/howdoi/Node.js%20CI?color=78dce8&label=Node.js%20CI&style=plastic` here